registrations/views.py

```diff
@@ -134,3 +138,19 @@ def get(self, request):
             )
             pending_user.delete()
             return HttpResponse(render_to_string('registration/validation-success.html'))
+
+class UserExternalTokenViewset(viewsets.ModelViewSet):
+    queryset = UserExternalToken.objects.exclude(is_disabled=True)
+    serializer_class = UserExternalTokenSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get_queryset(self):
+        return super().get_queryset().filter(user=self.request.user)
+    
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        instance.is_disabled = True
+        instance.save()
+        return Response(
+            status=status.HTTP_204_NO_CONTENT
+        )
```

Comment:
We don't need to delete, instead just disable. Which will be used when we implement blacklisting.